src/components/Citation.js

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Typography from '@material-ui/core/Typography';
+
+export default function Citation({ publication }) {
+  let citString = ""
+  let descriptor_names = [];
+  let full_names = [];
+  let last_names = [];
+  let out_file = "";
+  const authors = publication.authors
+
+  const populateAuthors = (publication) => {
+    const has_authors = publication.authors ? true : false;
+    if (has_authors) {
+      for (const author of authors) {
+        if (author.persona_firstname) {
+          descriptor_names.push(author.persona_lastname);
+          full_names.push(`${author.persona_firstname[0]}. ${author.persona_lastname}`)
+          last_names.push(author.persona_lastname)
+        } else {
+            descriptor_names.push(author.author_fullname);
```

Comment:
We should use `author_fullname` first, if available. Persona support was added later.

Reply:
I reworked most of the names strings so they should be more correct.  

According to the recent exports from the Midas Journal, the text citations don't have any quotes in them but all of the others have been updated

Reply:
Looking better.

Remaining issues:

- *BibTex* -> *BibTeX*.
- The file name for the bib should be something like *publication.bib*.
- There is still an wayward `"` in the XML.
- The Text citations should have quotes around the title (if this was wrong in the Midas Journal, it should still be corrected).
- The Text citations should have the URL and journal name.

Reply:
How does this look?

```
Kim E., Johnson H., Williams N. "Affine Transformation for Landmark Based Registration Initializer in ITK". The MIDAS Journal. 2011 Jul. http://hdl.handle.net/10380/3299
```

Reply:
Excellent!